src/Tests/ErrorHandlingTests.cs

```diff
@@ -35,5 +35,15 @@ public void Disabling_api_exceptions_should_not_throw_an_exception()
             service.ServiceConfig.ThrowExceptionOnAuthFailure = false;
             service.EmsSystem.Get();
         }
+
+        [Fact( DisplayName = "Requiring a call context but not providing one should throw" )]
+        public void Require_call_context_but_dont_provide_one()
+        {
+            var config = m_config.Clone();
+            config.RequireCallContext = true;
+            using var service = new EmsApiService( config );
+            Action causeFailure = () => service.EmsSystem.Get();
+            causeFailure.Should().Throw<EmsApiException>();
```

Comment:
Might be better to test your specific exception here?

Reply:
Yeah I had that initially and it did not work, but I was too lazy to dive into it, will do that next.